nixos/tests/borgbackup.nix

```diff
@@ -69,6 +71,7 @@ in {
             yearly = 5;
           };
           exclude = [ "*/${excludeFile}" ];
+          extraCreateArgs = [ "--exclude-caches" "--exclude-if-present" ".nobackup" ];
```

Comment:
I suppose to correctly test the feature the arguments should contain spaces or shell special characters

Reply:
Agreed. I will change `.nobackup` to `.dont backup`